.editorconfig

```diff
@@ -0,0 +1,6 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+indent_style = space
```

Comment:
Also `jsx` and `tsx`.

Is 2 a sensible default for _all_ files? If there's an exception we can put that in, rather than the other way around